src/platforms/dotnet/guides/aspnetcore/index.mdx

```diff
@@ -282,3 +282,25 @@ Opt-in to capture values from [System.Diagnostic.Activity](https://github.com/do
 - An [example](https://github.com/getsentry/sentry-dotnet/tree/main/samples/Sentry.Samples.AspNetCore.Mvc) using MVC and most of the SDK features. (**C#**)
 - An [example](https://github.com/sentry-demos/giraffe) using Giraffe on top of ASP.NET Core with a basic overview of available features. (**F#**)
 - For [more samples](https://github.com/getsentry/sentry-dotnet-samples) of the .NET SDKs.
+
+## Tunnel
+
+If events are being logged to sentry from the client-side then a [Tunnel](/platforms/javascript/troubleshooting/#using-the-tunnel-option) should be used.
+
+> A tunnel is an HTTP endpoint that acts as a proxy between Sentry and your application. Because you control this server, there is no risk of any requests sent to it being blocked.
```

Comment:
```suggestion
A tunnel is an HTTP endpoint that acts as a proxy between Sentry and your application. Because you control this server, there is no risk of any requests sent to it being blocked.
```